front/app/api/project_folder_images/types.ts

```diff
@@ -1,8 +1,23 @@
-import { API_PATH } from 'containers/App/constants';
-import streams, { IStreamParams } from 'utils/streams';
 import { TProjectFolderCardSize } from 'components/ProjectAndFolderCards/components/ProjectFolderCard';
+import projectFolderImagesKeys from './keys';
+import { Keys } from 'utils/cl-react-query/types';
+import { IStreamParams } from 'utils/streams';
 
-const apiEndpoint = `${API_PATH}/project_folders`;
+export type ProjectFolderImagesKeys = Keys<typeof projectFolderImagesKeys>;
+
+export type IQueryParameters = {
+  folderId: string;
+};
+
+export type IGetImagesQueryParameters = {
+  folderId: string;
+  streamParams?: IStreamParams | null;
```

Comment:
Let's not use streamParams as a namee anymore - you can just add image there, no?